Podcast/TextFieldSettingsTableViewCell.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  TextFieldSettingsTableViewCell.swift
+//  Podcast
+//
+//  Created by Drew Dunne on 10/22/17.
+//  Copyright © 2017 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+
+class TextFieldSettingsTableViewCell: SettingsTableViewCell {
+
+    var textField: UITextField!
+    let textFieldWidth: CGFloat = 30.0
+    let textFieldPadding: CGFloat = 12.0
+    let errorViewWidth: CGFloat = 17.0
+    let accessoryViewPadding: CGFloat = 8
+    
+    override init(style: UITableViewCellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        
+        textField = UITextField(frame: CGRect.zero)
+        textField.font = UIFont._14RegularFont()
+        textField.textColor = .offBlack
+        textField.autocapitalizationType = .none
+        textField.autocorrectionType = .no
+        addSubview(textField)
+
+        textField.snp.makeConstraints({ make in
+            make.left.equalTo(titleLabel.snp.right).offset(textFieldPadding)
```

Comment:
Always do leading and trailing instead of left and right because languages that read right to left can then use our app too!